examples/messengers/telegram/6_conditions_extras.py

```diff
@@ -0,0 +1,116 @@
+# %% [markdown]
+"""
+# 6. Conditions Extras
+
+This example shows how to use additional update filters
+inherited from the `pytelegrambotapi` library.
+"""
+
+# %%
+import os
+
+from dff.script import TRANSITIONS, RESPONSE, GLOBAL
+import dff.script.conditions as cnd
+from dff.messengers.telegram import (
+    PollingTelegramInterface,
+    TelegramMessenger,
+    TelegramMessage,
+    chat_join_request_handler,
+    my_chat_member_handler,
+    inline_handler,
+    message_handler,
+)
+from dff.pipeline import Pipeline
+from dff.utils.testing.common import is_interactive_mode
+
+
+# %% [markdown]
+"""
+In our Telegram module, we adopted the system of filters
+available in the `pytelegrambotapi` library.
+
+Aside from `message_handler` and `callback_query_handler`, you can use
+other triggers to interact with the api. In this example, we use
+handlers of other type as global conditions that trigger a response
+from the bot.
+
+Here, we use the following triggers:
+
+* `chat_join_request_handler`: join request is sent to the chat where the bot is.
+* `my_chat_member_handler`: triggered when the bot is invited to a chat.
+* `inline_handler`: triggered when an inline query is being sent to the bot.
+
+The other available conditions are:
+
+* `channel_post_handler`: new post is created in a channel the bot is subscribed to;
+* `edited_channel_post_handler`: post is edited in a channel the bot is subscribed to;
+* `shipping_query_handler`: shipping query is sent by the user;
+* `pre_checkout_query_handler`: order confirmation is sent by the user;
+* `poll_handler`: poll is sent to the chat;
+* `poll_answer_handler`: users answered the poll sent by the bot.
+
+You can read more on those in the Telegram documentation or in the docs for the `telebot` library.
+"""
+
+
+# %%
+# Like Telebot, TelegramMessenger only requires a token to run.
+# However, all parameters from the Telebot class can be passed as keyword arguments.
+messenger = TelegramMessenger(os.getenv("TG_BOT_TOKEN", "SOMETOKEN"))
+
+
+# %%
+script = {
+    GLOBAL: {
+        TRANSITIONS: {
+            ("greeting_flow", "node1"): cnd.any(
+                [
+                    # say hi when invited to a chat
+                    chat_join_request_handler(func=lambda x: True),
+                    # say hi when someone enters the chat
+                    my_chat_member_handler(func=lambda x: True),
+                ]
+            ),
+            # send a message when inline query is received
+            ("greeting_flow", "node2"): inline_handler(
+                func=lambda query: print(query) or query.query is not None
+            ),
```

Comment:
`print` I do not understand, why do we use `print` here

Reply:
It's look herd to understant, maybe it should replace by regular func instead lambda

Reply:
https://github.com/deeppavlov/dialog_flow_framework/pull/13/commits/3dd5610231c98e630af5b8619c654c19b63025ef